packages/react-instantsearch-hooks-server/src/getServerState.tsx

```diff
@@ -9,18 +8,21 @@ import {
 import type { InitialResults, InstantSearch } from 'instantsearch.js';
 import type { IndexWidget } from 'instantsearch.js/es/widgets/index/index';
 import type { ReactNode } from 'react';
+import type { renderToString as RenderToString } from 'react-dom/server';
```

Comment:
Isn't this type import subject to the same change (`react-dom/server` / `react-dom/server.js`) that makes us go with the dynamic import?

Reply:
the type import doesn't exist in the final build, as it's typescript only @dhayab :)